tardis/tests/tests_slow/test_w7.py

```diff
@@ -84,93 +56,86 @@ def setup(self):
                 self.obtained_radial1d_model.j_estimators)
 
     def test_j_blue_estimators(self):
         assert_allclose(
-                self.expected_ndarrays['j_blue_estimators'],
+                self.baseline['j_blue_estimators'],
                 self.obtained_radial1d_model.j_blue_estimators)
 
-        j_blues_norm_factor = self.expected_quantities['j_blues_norm_factor']
-        j_blues_norm_factor = j_blues_norm_factor * u.Unit('1 / (cm2 s)')
-
-        assert_allclose(
-                j_blues_norm_factor,
+        assert_quantity_allclose(
+                self.baseline['j_blues_norm_factor'],
                 self.obtained_radial1d_model.j_blues_norm_factor)
 
     def test_last_line_interactions(self):
         assert_allclose(
-                self.expected_ndarrays['last_line_interaction_in_id'],
+                self.baseline['last_line_interaction_in_id'],
                 self.obtained_radial1d_model.last_line_interaction_in_id)
 
         assert_allclose(
-                self.expected_ndarrays['last_line_interaction_out_id'],
+                self.baseline['last_line_interaction_out_id'],
                 self.obtained_radial1d_model.last_line_interaction_out_id)
 
         assert_allclose(
-                self.expected_ndarrays['last_line_interaction_shell_id'],
+                self.baseline['last_line_interaction_shell_id'],
                 self.obtained_radial1d_model.last_line_interaction_shell_id)
 
-        last_line_interaction_angstrom = self.expected_quantities['last_line_interaction_angstrom']
-        last_line_interaction_angstrom = last_line_interaction_angstrom * u.Unit('Angstrom')
-
-        assert_allclose(
-                last_line_interaction_angstrom,
+        assert_quantity_allclose(
+                self.baseline['last_line_interaction_angstrom'],
                 self.obtained_radial1d_model.last_line_interaction_angstrom)
 
     def test_nubar_estimators(self):
         assert_allclose(
-                self.expected_ndarrays['nubar_estimators'],
+                self.baseline['nubar_estimators'],
                 self.obtained_radial1d_model.nubar_estimators)
 
     def test_ws(self):
         assert_allclose(
-                self.expected_ndarrays['ws'],
+                self.baseline['ws'],
                 self.obtained_radial1d_model.ws)
 
-    def test_spectrum(self):
-        luminosity_inner = self.expected_quantities['luminosity_inner']
-        luminosity_inner = luminosity_inner * u.Unit('erg / s')
-
-        assert_allclose(
-                luminosity_inner,
+    def test_luminosity_inner(self):
+        assert_quantity_allclose(
+                self.baseline['luminosity_inner'],
                 self.obtained_radial1d_model.luminosity_inner)
 
-    def test_montecarlo_properties(self):
-        montecarlo_luminosity = self.expected_quantities['montecarlo_luminosity']
-        montecarlo_luminosity = montecarlo_luminosity * u.Unit('erg / s')
+    def test_spectrum(self):
```

Comment:
nope - I think we build up the structure first and then think about specific plots and tests.